src/assets/wise5/services/nodeService.ts

```diff
@@ -176,92 +177,87 @@ export class NodeService {
    * for how to choose a transition
    * @returns a promise that will return a transition
    */
-  chooseTransition(nodeId, transitionLogic): any {
+  protected chooseTransition(nodeId: string, transitionLogic: TransitionLogic): Promise<any> {
     const existingPromise = this.getChooseTransitionPromise(nodeId);
     if (existingPromise != null) {
       return existingPromise;
     }
     const promise = new Promise((resolve, reject) => {
       let transitionResult = this.getTransitionResultByNodeId(nodeId);
-      if (
-        transitionResult == null ||
-        (transitionLogic != null && transitionLogic.canChangePath == true)
-      ) {
+      if (transitionResult == null || transitionLogic.canChangePath == true) {
```

Comment:
I think you can delete the "== true" in the if conditional.

```suggestion
      if (transitionResult == null || transitionLogic.canChangePath) {
```